.github/workflows/dockerhub.yml

```
@@ -0,0 +1,22 @@
+name: Test Dockerhub Push
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Copy Repo Files
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Build and Publish Docker image to Dockerhub
+        uses: ./
+        with:
+          TAG: 'my-optional-tag-name'
+          DOCKERFILE_PATH: '.github/docker/Dockerfile'
+          BUILD_CONTEXT: '.'
+          DOCKERHUB_REPOSITORY: 'pinkrobin/gpr-docker-publish-example'
```

I am not sure this repo has authorization to push to this repo
```suggestion
          DOCKERHUB_REPOSITORY: 'hamelsmu/gpr-docker-publish-example'
```

---

Yes, that's true. This is just intended for testing purposes.